python/paddle/fluid/contrib/tests/README.md

```diff
@@ -0,0 +1,63 @@
+
+            calibrator = int8_utility.Calibrator( # Step 1
+                program=infer_program,
+                pretrained_model=model_path,
+                algo=algo,
+                exe=exe,
+                output=int8_model,
+                feed_var_names=feed_dict,
+                fetch_list=fetch_targets)
+
+* Sample data. Call the calibrator.sample_data() after executor run.
+            _, acc1, _ = exe.run(
+                program,
+                feed={feed_dict[0]: image,
+                      feed_dict[1]: label},
+                fetch_list=fetch_targets)
+
+                calibrator.sample_data() # Step 2
+
+* Save INT8 model. Call the calibrator.save_int8_model() after sampling over specified iterations (e.g., iterations = 50)
+            calibrator.save_int8_model() # Step 3
+			
+
+Please note that we assume FP32 pre-trained model has the `__model__ ` file with image and label feed variables which will be used by the accuracy measurememnt.
+
+## 2. How to run INT8 model
+You can load INT8 model by load_inference_model [API](https://github.com/PaddlePaddle/Paddle/blob/8b50ad80ff6934512d3959947ac1e71ea3fb9ea3/python/paddle/fluid/io.py#L991) and run INT8 inference similar as [FP32](https://github.com/PaddlePaddle/models/blob/develop/fluid/PaddleCV/object_detection/eval.py "FP32").
+
+        [infer_program, feed_dict,
+         fetch_targets] = fluid.io.load_inference_model(model_path, exe)
+		 
+
+## 3. Result
+We provide the results of accuracy and performance, measurd on [Intel® Xeon® Platinum 8180 Processor](https://ark.intel.com/products/120496/Intel-Xeon-Platinum-8180-Processor-38-5M-Cache-2-50-GHz- "Intel® Xeon® Platinum 8180 Processor") (also known as Intel® Xeon® Skylake8180).
+
+| Model  | Dataset  | FP32 Accuracy  | INT8 Accuracy  | Accuracy Diff  |
+| ------------ | ------------ | ------------ | ------------ | ------------ |
+| ResNet50  | Small  | 72.00%  | 72.00%  |  0.00% |
+| MobileNet-V1  | Small  | 62.00%  | 62.00%  | 0.00%  |
+| ResNet-50  | Full ImageNet Val  |  76.63%  | 76.14%  | 0.48% |
+| MobileNet-V1 | Full ImageNet Val  | 70.78%  | 70.41%  | 0.37%  |
+
+Please note that [Small](http://paddle-inference-dist.cdn.bcebos.com/int8/calibration_test_data.tar.gz "Small") is a subset of [full ImageNet validation dataset](http://www.image-net.org/challenges/LSVRC/2012/nnoupb/ILSVRC2012_img_val.tar "full ImageNet validation dataset"). Here is the typical dataset structure, similar to the requirement of [PaddlePaddle models](https://github.com/PaddlePaddle/models/tree/develop/fluid/PaddleCV/image_classification/data "PaddlePaddle models"):
+
+        $ ls data/ILSVRC2012/
+		val  val_list.txt
+
+| Model  | Batch Size  | FP32 Throughput  | INT8 Throughput  | Improvement  |
```

Comment:
Do you mean Throughput is (image/second)?

Reply:
Done